minseo/service/userService.js

```diff
@@ -0,0 +1,26 @@
+const userDao = require('../models/userDao');
+
+const signUp = async (name, email, password, profileImage) => {
+  const pwValidation = new RegExp(
+    '^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,20})'
+  );
+  const emailValidation = new RegExp(
+    '/^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i'
+  );
+  if (!pwValidation.test(password) && !emailValidation.test(email)) {
+    const err = new Error('PASSWORD_OR_EMAIL_IS_NOT_VALID');
+    err.statusCode = 409;
+    throw err;
+  }
+  const createUser = await userDao.createUser(
```

Comment:
- validation 조건이 둘 중 1개라도 조건을 만족하지 않으면 예외처리가 발동해야 되는 &&(and) 조건이 아닌 || (or)이 맞지 않을까요?
```suggestion
  );
  
  if (!pwValidation.test(password) || !emailValidation.test(email)) {
    const err = new Error('PASSWORD_OR_EMAIL_IS_NOT_VALID');
    err.statusCode = 409;
    throw err;
  }
  
  const createUser = await userDao.createUser(
```